assets/model_monitoring/components/model_monitor/model_monitor_feature_selector/spec.yaml

```diff
@@ -46,7 +46,6 @@ conf:
     channels:
       - defaults
     dependencies:
-      - python=3.8
```

Comment:
why did you remove python version from here? we recommend to specify the interpreter version across AzureML. EMS reject any environment to be registered with no python version specified